### PR TITLE
WFCORE-5497 Multiple syntax issues with 'wildfly-init-debian.sh'

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/WFCORE-5497

### CODE REVIEW

**Comment by parsharma** [comment]
@jfdenise could you please review this PR? There are four warnings that are left as the same left in this https://github.com/wildfly/wildfly-core/pull/4647.
Thanks

File: core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh [hunk 1]
```diff
@@ -78,14 +78,13 @@ if [ -z "$JBOSS_USER" ]; then
 fi
 
 # Check wildfly user
-id $JBOSS_USER > /dev/null 2>&1
-if [ $? -ne 0 -o -z "$JBOSS_USER" ]; then
+if ! id $JBOSS_USER > /dev/null 2>&1 || [ -z "$JBOSS_USER" ]; then
```
Comment (jfdenise) [inline core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh:81]: If you look at: https://github.com/wildfly/wildfly-core/pull/4647/files#diff-2365dc492fe86de1f4e9838f06559da1b1ef830bf6f3955f7b66484b90c7a6e5R73
the change is different although I guess the same tooling has been used to suggest changes. Correct?

Comment (parsharma) [reply core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh:81]: Yes, it was giving a warning on this line.

Comment (parsharma) [reply core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh:81]: I think I made mistake, I need to change the same line there as well. Thanks for noticing it.

Comment (parsharma) [reply core-feature-pack/common/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh:81]: Changes done.

**Comment by parsharma** [comment]
@jfdenise Could you please review this PR? Thanks

**Review by jfdenise — APPROVED** [review]

**Pull request merged**